palm/src/ctinfo.c

```diff
@@ -11,15 +11,20 @@
 #include "palm/memory.h"
 #include "palm/str.h"
 
+#define RED "\033[0;31m"
+#define LIGHT_PURPLE "\033[95m"
+#define RESET "\033[0m"
+
 static char *message_buffer = NULL;
 static int message_buffer_size = 0;
 static int message_line_length = 116;
 
 static char *abort_message_header = "abort: ";
-static char *error_message_header = "error: ";
-static char *warn_message_header = "warning: ";
+static char *error_message_header = "\033[0;31merror: \033[0m";
```

Comment:
You just defined globals for these exact colours, why not use them?

Reply:
These vars where changed before I added the defines, and it feels a little illegal to use them that way, but sure Ill change it